src/Typeahead.jsx

```diff
@@ -346,19 +365,24 @@ class Typeahead extends React.Component {
               onFocus={this.props.onFocus}
               value={this.state.currentValue}
               label={this.props.placeholder}
+              error={this.props.error}
```

Comment:
forgot to make a PR in TextInput before https://github.com/ship-components/ship-components-textinput/pull/2